### PR TITLE
Changes for avs/cvv changes

### PR DESCRIPTION
**Pull request by senthilkumar-muthusamy**
Changes for AVS CVV changes

### CODE REVIEW

File: etc/avs-cvv-settings.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        'tooltip' => __('Choose for which AVS result codes, the transaction must be auto reveresed', 'global-payments-gateway-provider-for-gravity-forms'),
+        'choices' => $avsDeclineCodes,
+        'default_value'     => array("N", "S", "U", "P", "R", "G", "C", "I"),
+        'multiple' => 'multiple',
+        'css'      => 'width: 450px'
+    ),
+    array(
+        'name' => 'cvn_reject_conditions[]',
+        'label' => __('CVN Reject Conditions', 'global-payments-gateway-provider-for-gravity-forms'),
+        'type' => 'select',
```
Comment (MSmedal) [inline etc/avs-cvv-settings.php:126]: image

Can we update this to look and operate like what we went with in WooCommerce? That way we can maintain consistent look and feel while also improving usability.

**Review by MSmedal — CHANGES_REQUESTED** [review]

File: etc/avs-cvv-settings.php (hunk 1 above)
Comment (senthilkumar-muthusamy) [reply etc/avs-cvv-settings.php:126]: As we discussed earlier the woocommerce classes are not working for this plugin. Need to find wordpress way to update the select box functionality

**Comment by senthilkumar-muthusamy** [comment]
@MSmedal can you please re review this PR. except the design of select box the functionality wise it's working fine.

**Comment by securesubmit-buildmaster** [comment]
> @MSmedal can you please re review this PR. except the design of select box the functionality wise it's working fine.

Agreed. If you'd like to update the version info and WP info I'll approve this.

**Review by MSmedal — APPROVED** [review]

**Pull request merged**